ground/gcs/src/plugins/config/configoutputwidget.cpp

```diff
@@ -102,18 +102,6 @@ ConfigOutputWidget::ConfigOutputWidget(QWidget *parent)
     connect(m_config->calibrateESC, &QAbstractButton::clicked, this,
             &ConfigOutputWidget::startESCCalibration);
 
-    // Track the ActuatorSettings object
-    addUAVObject("ActuatorSettings");
```

Comment:
Well, no point tracking an object unless you actually have a relation to it... which we do anyway.